src/command_modules/azure-cli-role/azure/cli/command_modules/role/_params.py

```diff
@@ -47,8 +48,14 @@ def load_arguments(self, _):
         c.argument('identifier', options_list=['--id'], help='identifier uri, application id, or object id of the application')
 
     with self.argument_context('ad app permission grant') as c:
-        c.argument('app_id', help='clientId of an existing app from which you want to grant permissions to your app')
-        c.argument('expires', help='Expiry date for the permissions in years, options include 1, 2 or never.')
+        c.argument('app_id', options_list=['--api', '--app-id'],
```

Comment:
This is somewhat atypical--having two long options for a single argument where one isn't being deprecated. Curious what your thoughts are here.

Reply:
You are right, deprecation is the right way. is it reasonable for lint rule to capture that?
Did some cross check, looks like the "--api-id" is not yet released, so I can just remove it